LICENSE

```diff
@@ -3416,3 +3416,13 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+This product bundles 'argparse' which is available under The MIT License.
```

Comment:
we are already using `cxxopts` library for the minificontroller, does `argparse` provide some extra features?

Reply:
One feature I cannot see in `cxxopts` that we used here with the `--docs` options is to provide variable number of parameters for an option. In `argparse` I could define `nargs` with 1 or 2 options depending on if the user wants to specify a second parameter for the PROCESSORS.md output. We also want to reuse the `--property` multiple times which looks to be a feature of `cxxopts` 3.0, but currently we are using 2.2. It seems that `cxxopts` is more lightweight, but with less features than `argparse` which looks to be in parity with the features of the python's argparse lib. I only saw the `ArgumentParser` class used in `EncryptConfig` so I replaced that, but I suppose it would be better to have a single argument parser in the overall codebase. What do you think, should we replace `cxxopts` in minificontroller too or can we use `cxxopts` with our options somehow?